install.sh

```diff
@@ -22,9 +22,13 @@ set -o pipefail  # Fail if one of the piped commands fails
 # Environment variables:
 #
 #   - LIQO_VERSION
-#     the version of Liqo to install. It can be a released version, a commit SHA or 'latest'.
+#     the version of Liqo to install. It can be a released version, a commit SHA or 'master'.
 #     Defaults to the latest released version.
 #
+#	- LIQO_DASHBOARD_VERSION
+#	  the version of the Liqo Dashboard to install. It can be a released version, a commit SHA or 'master'.
+#	  Defaults to the latest released version.
+#
```

Comment:
These lines should be dropped, since this is not a user-supplied environment variable

Reply:
Yes, sorry I forgot to get rid of them